app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :coments
```

Comment:
### **OPTIONAL**

- [ ] Please fix this typo error for `comments`

Reply:
@CollinsTatang Thanks for pointing this out I will be  fixing it right away!